### ISSUE TITLE
 Closes the connection temporary stream:error

### ISSUE THREAD

**Issue by driesken** [jitsi/jicofo#6]
I've just retrieved the newest jitsi meet by git and have installed jicofo (Redhat) since it seems to be required. The build using 'ant' was successfull.

I've followed the steps in the howto and it seems to work; when I join the channel, I become a moderator. However, the video stream doesn't work. Jicofo was started with the following parameters:
./jicofo.sh --host=localhost --domain=bl17-15-test.xxxxxxxx.be --port=5447 --secret=qgN2Nau4jA9REbSMf9z2 --user_domain=auth.bl17-15-test.xxxxxxxx.be --user_name=focus --user_password=blabla

Note: port 5447 is the component_port in my prosody.

It appears that there are a lot of 'Failed to connect' errors.

`16:05:32.566 FINE: [9] plugin.reconnectplugin.ReconnectPluginActivator.start().238 Found 0 already installed providers.
16:05:33.124 SEVERE: [22] org.jivesoftware.smack.PacketReader.notifyConnectionError() Closes the connection temporary
stream:error (text)
        at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:264)
        at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:45)
        at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:74)
16:05:36.984 INFO: [25] org.jitsi.jicofo.xmpp.FocusComponent.handleIQSet().222 Focus request for room: test2@conference.bl17-15-test.xxxxxxxx.be
16:05:36.993 INFO: [25] org.jitsi.jicofo.FocusManager.createConference().230 Created new focus for test2@conference.bl17-15-test.xxxxxxxx.be@auth.bl17-15-test.xxxxxxxx.be conferences count: 1 options:
    channelLastN: -1
    adaptiveLastN: false
    adaptiveSimulcast: false
    bridge: jitsi-videobridge.bl17-15-test.xxxxxxxx.be
    enableFirefoxHacks: false
    openSctp: true
16:05:37.153 SEVERE: [28] org.jivesoftware.smack.PacketReader.notifyConnectionError() Closes the connection temporary
stream:error (text)
        at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:264)
        at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:45)
        at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:74)
16:05:52.891 INFO: [18] org.jitsi.jicofo.FocusManager.expireLoop().458 Focus idle timeout for test2@conference.bl17-15-test.xxxxxxxx.be
16:05:52.891 INFO: [18] org.jitsi.impl.protocol.xmpp.OperationSetColibriConferenceImpl.expireConference().334 Nothing to expire - no conference allocated yet
16:05:52.891 SEVERE: [18] org.jitsi.jicofo.JitsiMeetConference.leaveTheRoom().360 Chat room already left!
16:05:52.987 SEVERE: [19] org.jitsi.jicofo.util.RegisterThread.run().42 net.java.sip.communicator.service.protocol.OperationFailedException: Failed to connect
net.java.sip.communicator.service.protocol.OperationFailedException: Failed to connect
        at org.jitsi.impl.protocol.xmpp.XmppProtocolProvider.register(XmppProtocolProvider.java:161)
        at org.jitsi.jicofo.util.RegisterThread.run(RegisterThread.java:38)
Caused by: No response from the server.:
        at org.jivesoftware.smack.NonSASLAuthentication.authenticate(NonSASLAuthentication.java:73)
        at org.jivesoftware.smack.XMPPConnection.login(XMPPConnection.java:217)
        at org.jitsi.impl.protocol.xmpp.XmppProtocolProvider.register(XmppProtocolProvider.java:156)
        ... 1 more
16:05:57.002 SEVERE: [26] org.jitsi.jicofo.util.RegisterThread.run().42 net.java.sip.communicator.service.protocol.OperationFailedException: Failed to connect
net.java.sip.communicator.service.protocol.OperationFailedException: Failed to connect
        at org.jitsi.impl.protocol.xmpp.XmppProtocolProvider.register(XmppProtocolProvider.java:161)
        at org.jitsi.jicofo.util.RegisterThread.run(RegisterThread.java:38)
Caused by: No response from the server.:
        at org.jivesoftware.smack.NonSASLAuthentication.authenticate(NonSASLAuthentication.java:73)
        at org.jivesoftware.smack.XMPPConnection.login(XMPPConnection.java:217)
        at org.jitsi.impl.protocol.xmpp.XmppProtocolProvider.register(XmppProtocolProvider.java:156)
        ... 1 more
16:05:57.153 INFO: [18] org.jitsi.impl.protocol.xmpp.XmppProtocolProvider.info() XMPP provider Jabber:focus@auth.bl17-15-test.xxxxxxxx.be/focus8924008725472057@localhost disconnected
16:05:57.155 INFO: [18] org.jitsi.jicofo.FocusManager.conferenceEnded().252 Disposed conference for room: test2@conference.bl17-15-test.xxxxxxxx.be conference count: 0
16:09:10.815 INFO: [34] org.jitsi.jicofo.xmpp.FocusComponent.handleIQSet().222 Focus request for room: test2@conference.bl17-15-test.xxxxxxxx.be
16:09:10.815 INFO: [34] org.jitsi.jicofo.FocusManager.createConference().230 Created new focus for test2@conference.bl17-15-test.xxxxxxxx.be@auth.bl17-15-test.xxxxxxxx.be conferences count: 1 options:
    channelLastN: -1
    adaptiveLastN: false
    adaptiveSimulcast: false
    bridge: jitsi-videobridge.bl17-15-test.xxxxxxxx.be
    enableFirefoxHacks: false
    openSctp: true
16:09:10.976 SEVERE: [37] org.jivesoftware.smack.PacketReader.notifyConnectionError() Closes the connection temporary
stream:error (text)
        at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:264)
        at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:45)
        at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:74)
16:09:27.162 INFO: [18] org.jitsi.jicofo.FocusManager.expireLoop().458 Focus idle timeout for test2@conference.bl17-15-test.xxxxxxxx.be
16:09:27.163 INFO: [18] org.jitsi.impl.protocol.xmpp.OperationSetColibriConferenceImpl.expireConference().334 Nothing to expire - no conference allocated yet
16:09:27.163 SEVERE: [18] org.jitsi.jicofo.JitsiMeetConference.leaveTheRoom().360 Chat room already left!
16:09:30.820 SEVERE: [35] org.jitsi.jicofo.util.RegisterThread.run().42 net.java.sip.communicator.service.protocol.OperationFailedException: Failed to connect
net.java.sip.communicator.service.protocol.OperationFailedException: Failed to connect
        at org.jitsi.impl.protocol.xmpp.XmppProtocolProvider.register(XmppProtocolProvider.java:161)
        at org.jitsi.jicofo.util.RegisterThread.run(RegisterThread.java:38)
Caused by: No response from the server.:
        at org.jivesoftware.smack.NonSASLAuthentication.authenticate(NonSASLAuthentication.java:73)
        at org.jivesoftware.smack.XMPPConnection.login(XMPPConnection.java:217)
        at org.jitsi.impl.protocol.xmpp.XmppProtocolProvider.register(XmppProtocolProvider.java:156)
        ... 1 more
16:09:30.970 INFO: [18] org.jitsi.impl.protocol.xmpp.XmppProtocolProvider.info() XMPP provider Jabber:focus@auth.bl17-15-test.xxxxxxxx.be/focus8924222548382055@localhost disconnected
16:09:30.971 INFO: [18] org.jitsi.jicofo.FocusManager.conferenceEnded().252 Disposed conference for room: test2@conference.bl17-15-test.xxxxxxxx.be conference count: 0`

**Comment by driesken** [comment]
Found the problem. When you have "c2s ports" in prosody set to anything else than 5222, the above errors are shown. I tried to solve it by replacing "5222" in source with "5322" (port in my case).

This seems to work, however, a new error is thrown:

21:47:42.666 INFO: [36] org.jitsi.protocol.xmpp.AbstractOperationSetJingle.sendAddSourceIQ().277 Notify add SSRCtest2@conference.bl17-15-test.xxxxxxxx.be/01896112 SID: 6qms6r8unk08d

21:47:42.811 SEVERE: [36] org.jitsi.impl.protocol.xmpp.OperationSetJingleImpl.accept().95 Received an error: code=500 message=null
